modules/oe_theme_helper/tests/src/Functional/NodeViewRoutesMetadataTest.php

```diff
@@ -68,18 +69,26 @@ public function setUp(): void {
       'view latest version',
       'view any unpublished content',
     ]);
+
+    FilterFormat::create([
```

Comment:
Let's remove this.

Reply:
I thought it doesn't work without that.

Reply:
Without creating filter format test is not valid.